config/confd/templates/env.tmpl

```diff
@@ -64,6 +64,10 @@ REDIS_LOGS_AUTH={{getenv "REDIS_LOGS_AUTH"}}
 REDIS_LOGS_RO_HOST={{getenv "REDIS_LOGS_RO_HOST"}}
 REDIS_LOGS_RO_AUTH={{getenv "REDIS_LOGS_RO_AUTH"}}
 REDIS_LOGS_SHARDED_PUBSUB={{getenv "REDIS_LOGS_SHARDED_PUBSUB"}}
+LOGS_REDIS_READ_PCT={{getenv "LOGS_REDIS_READ_PCT"}}
```

Comment:
Can we follow the convention of `REDIS_LOGS` instead of `LOGS_REDIS` used in other places?